run-linter.sh

```diff
@@ -0,0 +1,67 @@
+directories=""
+separate_files="sqs_status.py"
+fail=0
+
+function prepare_venv() {
+    virtualenv -p python3 venv && source venv/bin/activate && python3 `which pip3` install pycodestyle
+}
+
+echo "----------------------------------------------------"
+echo "Running Python linter against following directories:"
+echo $directories
+echo "----------------------------------------------------"
+echo
+
+ls -1
```

Comment:
This is debugging left-over. Pavel has already removed it from the other scripts.

Reply:
Fixed 😉